packages/orbit-components/src/Loading/index.tsx

```diff
@@ -1,194 +1,76 @@
-  ${({
-    type,
-    theme,
-    customSize,
-  }: {
-    type: Props["type"];
-    theme: typeof defaultTheme;
-    customSize?: Props["customSize"];
-  }) => css`
-    position: ${type === TYPE_OPTIONS.BUTTON_LOADER && "absolute"};
-    top: ${type === TYPE_OPTIONS.BUTTON_LOADER && "0"};
-    ${left}: ${type === TYPE_OPTIONS.BUTTON_LOADER && "0"};
-    width: ${type === TYPE_OPTIONS.BUTTON_LOADER && "100%"};
-    ${getHeight({ type, customSize })};
-    padding: ${type !== TYPE_OPTIONS.INLINE_LOADER && theme.orbit.paddingLoading};
-    display: ${type === TYPE_OPTIONS.INLINE_LOADER ? "inline-flex" : "flex"};
-    flex-direction: ${type === TYPE_OPTIONS.PAGE_LOADER ? "column" : "row"};
-    justify-content: ${getAlign({ type })};
-    align-items: center;
-    overflow: hidden;
-    box-sizing: border-box;
-  `}
-`;
-
-StyledLoading.defaultProps = {
-  theme: defaultTheme,
-};
-
-const StyledLoadingText = styled.div<{ type: Type }>`
-  ${({ theme, type }) => css`
-    font-family: ${theme.orbit.fontFamily};
-    font-size: ${theme.orbit.fontSizeTextNormal};
-    color: ${theme.orbit.colorTextLoading};
-    line-height: ${theme.orbit.lineHeightTextNormal};
-    margin-top: ${type === TYPE_OPTIONS.PAGE_LOADER && theme.orbit.spaceMedium};
-    margin-${left}: ${type !== TYPE_OPTIONS.PAGE_LOADER && theme.orbit.spaceSmall};
-  `};
-`;
-
-StyledLoadingText.defaultProps = {
-  theme: defaultTheme,
-};
-
-export const StyledSpinner = styled.svg<{ customSize?: Props["customSize"] }>`
-  ${({ customSize }) => css`
-    width: ${customSize ? `${customSize}px` : "40px"};
-    height: ${customSize ? `${customSize}px` : "40px"};
-    animation: ${SpinnerAnimation} 0.75s linear infinite;
-  `};
-`;
-
-const StyledSpinnerCircle = styled.circle<{ customSize?: Props["customSize"] }>`
-  ${({ type, theme, customSize }) => css`
-    fill: transparent;
-    stroke: ${type === TYPE_OPTIONS.BUTTON_LOADER ? "currentColor" : theme.orbit.paletteCloudDark};
-    stroke-width: 3px;
-    stroke-linecap: round;
-    stroke-dasharray: ${customSize ? `${customSize * 3 + 8}px` : "128px"};
-    stroke-dashoffset: ${customSize ? `${customSize + 24}px` : "64px"};
-  `};
-`;
-
-StyledSpinnerCircle.defaultProps = {
-  theme: defaultTheme,
-};
-
-const StyledLoader = styled.div`
-  display: flex;
-  justify-content: center;
-  align-items: center;
-`;
-
-const StyledLoaderCircle = styled.div`
-  width: 8px;
-  height: 8px;
-  border-radius: 50%;
-  margin-${right}: 6px;
-  background: ${({ theme }) => theme.orbit.paletteCloudDark};
-  animation: ${LoaderAnimation} 1.25s infinite ease-in-out;
-  &:nth-child(2) {
-    animation-delay: 0.1s;
-  }
-  &:nth-child(3) {
-    animation-delay: 0.2s;
-    margin: 0;
-  }
-`;
-
-StyledLoaderCircle.defaultProps = {
-  theme: defaultTheme,
+const CircleLoader = ({ animationDelay }: { animationDelay?: string }) => {
+  return (
+    <div
+      className="animate-loader rounded-circle bg-cloud-dark h-xs w-xs me-[6px] [&>*:nth-child(3)]:my-0"
```

Comment:
```suggestion
      className="animate-loader rounded-circle bg-cloud-dark h-xs w-xs me-[6px] [&:nth-child(3)]:m-0"
```

You were targeting descendants with the nth(3) property, but you wanna target the actual element. If it is the third child of its parent, it should have `margin: 0` (and not `margin-y: 0` 😉).